pkg/kudoctl/cmd/package_list_params.go

```diff
@@ -108,25 +108,25 @@ func displayParamsTable(pf *packages.Files, out io.Writer, printRequired, printN
 			}
 		}
 		if found {
-			fmt.Fprintln(out, table)
+			_, _ = fmt.Fprintln(out, table)
```

Comment:
I think we should rather handle the errors for fprintln and return them immediately rather than producing inconsistent output?

Reply:
I was mostly going to fix the warnings, and i'm not sure there can actually be errors produced, but yes, you're right. I've added error handling and return if an error is return.